.github/workflows-source/ci-main.yml

```diff
@@ -368,7 +362,7 @@ jobs:
 
   cargo-clippy-linux:
     name: Cargo Clippy Linux
-    <<: *dind-large-setup
```

Comment:
I'd leave this on `dind-large-setup`. It takes quite some resources so it's safer to have it on `dind-large-setup`.

Reply:
Please ignore this PR, I was testing something out before, then reused it to test the slack integration

Reply:
@cgundy test.

Reply:
@dfinity/idx test